apps/helloworld/quarkus/dockerpush_docker.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# use docker images | grep quarkus to get the image ID for $1
+
+docker login docker.io
+
+docker tag $1 docker.io/burrsutter/quarkus-demo:2.0.0
+
+docker push docker.io/burrsutter/quarkus-demo:2.0.0
+
```

Comment:
same comment as above

Reply:
For this one I leave as it since I'm not sure that has RHD org we have a docker.io repo and we want to push quay